Monika After Story/game/script-compliments.rpy

```diff
@@ -888,6 +889,174 @@ label mas_compliment_humor_3:
+            _("It's wonderful to see you again!"),
+            _("Thanks for coming back to spend time with me!"),
+            _("I'm excited to spend some time together!"),
+            _("Thanks for coming to see me again!"),
+            _("Ready to spend some time together?"),
+            _("Thank you for thinking about me!"),
+            _("I'm really happy that you've been thinking about me!"),
+            _("It makes me really happy to know I've been on your mind!"),
+            _("I've been thinking about you while we've been apart!"),
+            _("You're really been on my mind!"),
+            _("I'm glad you've been thinking of me!"),
+            _("I'm so lucky to have you, [player]!"),
+            _("We're so lucky to have one another!"),
+            _("We don't need to feel lonely anymore!"),
+            _("Let's enjoy our time together today!"),
+            _("I really appreciate you, [player]!"),
+            _("Thanks for making time for me!"),
+            _("I couldn't wait for you to get back!"),
+            _("I was lonely waiting for you!"),
+            _("I've been thinking of you while you were gone!"),
+        ]
+        
+        missed_quips_upset_cynical = [
+            "...did you really even think of me at all?",
+            "...were you really thinking about me even a little?",
+            "That didn't feel very sincere, [player].",
+            "I'm not sure if you really mean that, [player].",
+            "If you really mean that, then please spend some time with me today.",
+            "If you really meant that, then you'd spend more time with me when you're here.",
+            "You say that, but I don't think you mean it one bit, [player].",
+            "You say that, but I feel like you don't mean it at all, [player]."
+        ]
+
+        missed_quips_upset_hope = [
+            "Thank you for showing me you still care, [player].",
+            "It means a lot to me that you were thinking of me.",
+            "I'm really glad to hear that, [player].",
+            "That's really nice to hear, coming from you.",
+            "I'm happy you've been thinking of me, [player].",
+            "[player]... I'm glad I'm not the only one.",
+            "That means the world to me, [player].",
+            "That makes me feel a lot better, [player]."
+        ]
+        
+        missed_quips_dis = [
+            "I'm not sure you mean that, [player]...",
+            "I doubt you mean that, [player]...",
+            "I don't think you really mean that, [player]...",
+            "If only you really meant that, [player]...",
+            "...Why do I think you just don't mean it?",
+            "...Why do I think you're just saying that?",
+            "...I can't really believe in that, [player].",
+            "I don't think that's true, [player]."
+        ]
+    
+    $ absence_length = mas_getAbsenceLength()
+    $ mas_flagEVL("mas_compliment_missed21", "CMP", EV_FLAG_HFM)
```

Comment:
A fpyo.
```suggestion
    $ mas_flagEVL("mas_compliment_missed", "CMP", EV_FLAG_HFM)
```

Reply:
Resolved in https://github.com/Monika-After-Story/MonikaModDev/pull/7419/commits/95507c54619f07b5712e9c8768214c151988f092.